--- README.orig	Wed May 29 16:06:52 2002
+++ README	Fri Oct  4 20:36:35 2002
@@ -12,6 +12,7 @@
         Mac{Roman,CentralEurope,Iceland,Croatian,Romania},
         Mac{Cyrillic,Ukraine,Greek,Turkish},
         Macintosh
+	ANSEL
     Semitic languages
         ISO-8859-{6,8}, CP{1255,1256}, CP862, Mac{Hebrew,Arabic}
     Japanese
@@ -132,15 +133,22 @@
     recompiled. Just set the LD_PRELOAD environment variable, that's it!
 
 
-Distribution:
+Distribution of original package (libiconv):
     ftp://ftp.gnu.org/pub/gnu/libiconv/libiconv-1.8.tar.gz
     ftp://ftp.ilog.fr/pub/Users/haible/gnu/libiconv-1.8.tar.gz
 
-Homepage:
+Homepage of original package (libiconv):
     http://www.gnu.org/software/libiconv/
 
+Distribution of modified package (libiconv-gedcom):
+    http://sourceforge.net/projects/gedcom-parse
+
 Bug reports to:
     <bug-gnu-libiconv@gnu.org>
+    For specific ANSEL bugs, please use the web interface at SourceForge
+    (link given above)
 
 
 Bruno Haible <brunoe@clisp.org>
+
+Modifications: Peter Verthez <peter.verthez@advalvas.be>
